sdk/include/ndk/kefuncs.h

```diff
@@ -33,6 +33,54 @@ extern "C"
 
 #ifndef NTOS_MODE_USER
 
+//
+// Affinity helper
```

Comment:
```suggestion
// Affinity helpers
```
(if we also count the bit-helpers below)

Reply:
fixed